cmd/swarm/import.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2016 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/cmd/utils"
+
+	"github.com/ethereum/go-ethereum/swarm/storage"
+
+	"github.com/ethereum/go-ethereum/log"
+	"gopkg.in/urfave/cli.v1"
+)
+
+func importdb(ctx *cli.Context) {
+	args := ctx.Args()
+	if len(args) != 2 {
+		utils.Fatalf("need two arguments <source-db-dir> <target-db-dir")
+	}
+
+	sourcedir := args[0]
+	targetdir := args[1]
+
+	chunkcount, err := storage.Import(sourcedir, targetdir)
+	if err != nil {
+		utils.Fatalf("import failed: %s", err)
+	}
+
+	log.Trace(fmt.Sprintf("%d chunks imported", chunkcount))
```

Comment:
This should be `log.Info` so you don't need to set `--verbosity` in order to see it.